scripts/blog_template.py

```diff
@@ -1,12 +1,46 @@
 import os
 import sys
 import pandas as pd
+import argparse
 from jinja2 import Environment, FileSystemLoader
 
 script_dir = os.path.dirname(__file__)
 
+# a mapping of column names from the MS Form to names
+# used in the jinja template
+col_dict = {
```

Comment:
You'll need to update this mapping if you change the wording of the questions in the MS Form